cogs/misc.py

```diff
@@ -1,16 +1,58 @@
 import random
+import time
+from datetime import datetime
 
 import discord
-from discord.ext import commands
+from discord import Embed
+from discord.ext import commands, tasks
 
 from cogs.utils import Colour, is_trusted
 
+import aiohttp
+
 
 class Misc(commands.Cog):
     """Miscellaneous code and features, for the fun of it."""
 
     def __init__(self, bot):
         self.bot = bot
+        self.proton_pw = None
+        self.query_protondb.start()
+
+    @tasks.loop(hours=1.0)
+    async def query_protondb(self):
+        """Query the ProtonDB api for Project Winter data once every 2 hours"""
+
+        self.proton_pw = dict()
```

Comment:
Can you do this after requesting the data?

Reply:
How do you mean after requesting the data, shouldn't clearing the dict with the previous data come before the task?

Reply:
Not really, clear the dictionary once you have the new data. It doesn't hurt if we have a little old data for a bit - but having an empty dictionary does hurt.